### ISSUE TITLE
Leveling: Add ship filters

### ISSUE THREAD

**Issue by dragonjet** [KC3Kai/KC3Kai#1035]
https://www.facebook.com/kc3kai/photos/a.1649601521994007.1073741828.1649102015377291/1655109144776578/?type=3&comment_id=1655230731431086&offset=0&total_comments=5&comment_tracking=%7B%22tn%22%3A%22R%22%7D

**Comment by LostLogia4** [comment]
Then there's several more functions to add: 
[] Displaying ship names
[] Option to load current/preset fleet

Also, lel DJ u display Umikaze in a place that are still salty over last event's e4.

**Comment by NicolaosGreene** [comment]
Honestly I found e4 more vexing than e7 last event, but yes the current leveling gets difficult to use if you have a large number of ships without anyway to filter/search for them

**Comment by Nya-chan** [comment]
This should now be complete with PR #1850, though @LostLogia4's additions might be nice to add as well... not sure how to incorporate that, make a new issue?

**Comment by Javran** [comment]
@Nya-chan ship names are already displayed, that leaves just an option to load current/preset fleet. My suggestion is to just focus on current fleet as preset fleet can be loaded to be current fleet anyway - and for that I can think of an easy solution: just adding another section "Ships in Current Fleet" or something shorter. when ships falls into "Other Ships" section, we check if the ship in question is in a fleet then move her to this new section.

**Comment by Nya-chan** [comment]
@Javran I agree about the current/preset fleets. Should I create an issue for this alone?

**Comment by Javran** [comment]
@Nya-chan I'm good with either way. slightly preferred if you just leave this open, that's just for  keeping all context in one issue.

**Comment by Nya-chan** [comment]
OK, will just add this on my list of saved issues to work on, then :)
